tests/test_preprints.py

```diff
@@ -582,9 +580,8 @@ def test_approve_withdrawal_request_pre_moderated_preprint(
         assert PreprintDetailPage(driver, verify=True)
         WebDriverWait(driver, 5).until(EC.visibility_of(preprint_page.title))
         assert preprint_page.title.text == preprint_title
-        assert (
-            preprint_page.status_explanation.text == 'This preprint has been withdrawn.'
-        )
+        # Add assert for "This preprint has been withdrawn" after [ENG-5092] is fixed.
```

Comment:
Leave this code snippet in our codebase. Just comment it out for future reference. 

`# assert preprint_page.status_explanation.text == 'This preprint has been withdrawn.'
`